### PR TITLE
Working with sbt's mutability to publish snapshots to our internal nexus

### PR DESCRIPTION
**Pull request by adamdecaf**
I think this'll allow us to publish snapshots to our internal nexus and then releases to bintray.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -15,25 +15,22 @@ resolvers ++= Seq(
...
+  repository in bintray := "oss",
+  licenses ++= Seq(("Apache-2.0", url("http://www.apache.org/licenses/LICENSE-2.0.html")))
+)
+
 publishTo <<= (version) { v =>
   if (v.trim.endsWith("SNAPSHOT")) {
     Some("Banno Snapshots Repo" at "http://nexus.banno.com/nexus/content/repositories/snapshots")
   } else {
-    Some("Banno Releases Repo" at "http://nexus.banno.com/nexus/content/repositories/releases")
+    None
```
Comment (trentio) [inline build.sbt]: Here dont you want to just point it at the bintray.

**Comment by rubbish** [comment]
Will this break publishing releases to bintray?

**Comment by adamdecaf** [comment]
@rubbish @trentio yea, I found a fix.

**Comment by trentio** [comment]
:+1:

**Comment by tbrown1979** [comment]
:+1:

**Pull request merged**